uikit/common/sdk/src/main/java/com/sygic/ui/common/sdk/utlils/AddressFormatUtils.kt

```diff
@@ -0,0 +1,52 @@
+package com.sygic.ui.common.sdk.utlils
+
+import android.text.TextUtils
+import com.sygic.ui.common.sdk.data.PoiData
+
+fun getAddressTitle(poiData: PoiData): String {
+    poiData.name?.let { if (it.isNotEmpty()) return it }
+    poiData.street?.let { if (it.isNotEmpty()) return getStreetWithHouseNumber(it, poiData.houseNumber) }
+    poiData.city?.let { if (it.isNotEmpty()) return getCityWithPostal(it, poiData.postal) }
+
+    return getFormattedLocation(poiData.coordinates)
+}
+
+fun getAddressSubtitle(poiData: PoiData): String {
+    poiData.name?.let { if (it.isNotEmpty()) return getStreetWithHouseNumberAndCityWithPostal(poiData) }
+    poiData.street?.let { street ->
+        if (street.isNotEmpty()) poiData.city?.let { if (it.isNotEmpty()) return getCityWithPostal(it, poiData.postal) }
```

Comment:
tato podmienka je spravna? ak `street` nie je empty, nepouziju ju, ale pouzi `city`?

Reply:
ano, je to závislé na tom, co se dalo do title. Ale předělám to, aby to dávalo větší smysl.